wom/src/main/resources/reference.conf

```diff
@@ -4,3 +4,10 @@ yaml {
   # The maximum nested depth for nodes (sequences + mappings) that will be parsed in a YAML
   max-depth = 1000
 }
+
+womParse {
```

Comment:
Should be `wom-parse`.

> Config keys are encouraged to be hyphen-separated rather than camelCase.

https://github.com/lightbend/config/blob/master/HOCON.md#hyphen-separated-vs-camelcase

Reply:
This makes sense. 

The root problem being solved here, has to do with nested WDL scatters. The current WDL compiler generates a sub-workflow for the inner loop. This causes problems for [dxWDL](https://github.com/dnanexus/dxWDL), which uses the resulting WOM data structures. 
 
I had two questions:
1. Which document should I add to in `docs`?
2. Is there a test similar to what you are suggesting? What is a good example to start from?